packages/core/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@stitches/core",
   "version": "1.3.1-1",
   "description": "The modern CSS-in-JS library",
-  "type": "module",
```

Comment:
I don't think that you actually needed this. You still provide `exports.import` after all. Having those packages as modules changes how `.js` (and this `.d.ts`) files are treated and supporting types with this setup would be much more complicated. So I think that there is no harm if we just remove this from here to make `types` work.